Edit javascript codebyte1 (#2245)
Codecademy/docs: content/javascript/concepts/dates/terms/getHours/getHours.md

  - 'paths/front-end-engineer-career-path'
---

The **`.getHours()`** method returns the hours of a date according to the local time.

## Syntax

> ```js

Reviewer: ```suggestion
```shell
```

myDate.getHours();
```

> The return value of `getHours()` is an integer number between `0` and `23`, meaning the hour according to the local time.

Reviewer: ```suggestion
The return value of `.getHours()` is an integer number between `0` and `23`, meaning the hour according to the local time.
```

// Output: It is not 10 in the morning now.
```

## Codebyte Example

> The example below is runnable, and first, it defines a variable called `currentTime` with a new `Date` of today's date and time, then uses the `getHours()` method to store the hour from that date into a variable called `currentHour`. Then `console.log` prints the number representing that hour of the day to the screen:

Reviewer: ```suggestion
The example below is runnable, and first, it defines `currentTime` with today's date and time, then the result of the `.getHours()`method is saved in `currentHour`. Lastly, a `console.log` call prints the value to the screen:
```

```codebyte/javascript
const currentTime = new Date();
const currentHour = currentTime.getHours();

> console.log("The current hour of the day is: " + currentHour);

Reviewer: ```suggestion
console.log("The current hour is: " + currentHour);
```
